src/db/models/Product.model.ts

```diff
@@ -28,6 +28,7 @@ export interface ProductAttributes {
 	category_id?: number;
 	manufacture_id: number;
 	unit_id?: number;
+	user_id?: number | void;
```

Comment:
Explain this change please

Reply:
The user_id field was not described in the product model, after I described it ts said it was already used somewhere, I found it in the code and there it has type number | void. Apparently someone before me described it. And in order not to break anything I had to describe it the same way.